### ISSUE TITLE
Missing Datasets used for Metric Generation and Sim Reproduction

### ISSUE THREAD

**Issue by ameyalambat128** [IRMV-Manipulation-Group/RL-GSBridge#5]
Hello,

I am trying to replicate the results from the paper associated with this repository. However, I noticed that the datasets used to generate the metrics in the paper are not available to access or download. Could you please provide access to these datasets or clarify how they can be obtained?

This would greatly help in reproducing the results and further research.

Thank you for your time and assistance. 

<img width="724" alt="Image" src="https://github.com/user-attachments/assets/1dbb4f45-9106-4833-aeeb-b81cc47c3d50" />

**Comment by FurryGreen** [comment]
Hello,
Thank you for your interest in our work! We have released some of the steps related to GS reconstruction and provided a portion of the dataset used in the paper. You can refer to the **An Example** section under **Real2Sim Reconstruction** to reproduce the results.

For reconstructing your own data using our method, we plan to release additional guidelines and resources in the future. 

Feel free to reach out if you have further questions.

**Comment by FurryGreen** [comment]
The simulation training code and reproduction steps will be available by mid-April. Thank you for your patience and support!
